tools/resourcedocsgen/cmd/docs.go

```diff
@@ -0,0 +1,201 @@
+package cmd
```

Comment:
This file is basically everything that was in `main.go`. It's now specific to generating the resource docs from the unmarshalled main `PackageSpec`.